systers_portal/static/js/search_menu.js

```diff
@@ -0,0 +1,62 @@
+// Extracting cookies
+function getCookie(name) {
+            var cookieValue = null;
+            if (document.cookie && document.cookie !== "") {
+                var cookies = document.cookie.split(";");
+                cookies.forEach(function(cookie){
+					cookie = jQuery.trim(cookie);
+                    // Does this cookie string begin with the name we want?
+                    inner:if (cookie.substring(0, name.length + 1) === (name + "=")) {
+                        cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+                        break inner;
+                    }
+                });
+            }
+            return cookieValue;
+        }
```

Comment:
Please provide an explanation concisely in the comments why we need to extract cookies.

Reply:
Done